cmd/kube-apiserver/app/server.go

```diff
@@ -557,6 +573,35 @@ func buildGenericConfig(
 	return
 }
 
+// BuildAuthenticator constructs the authenticator
+func BuildAuthenticator(s *options.ServerRunOptions, EgressSelector *egressselector.EgressSelector, extclient clientgoclientset.Interface, versionedInformer clientgoinformers.SharedInformerFactory) (authenticator.Request, *spec.SecurityDefinitions, error) {
```

Comment:
@marun something got mixed up here. This func is neither used here, nor necessary.

Reply:
Done